frontend/src/pages/RoomPage/components/CollabContainer/components/PeerPrepCodeMirror.tsx

```diff
@@ -100,6 +101,28 @@ const PeerPrepCodeMirror = () => {
     }
   };
 
+  const submitCode = (src :string) => {
```

Comment:
Actually, I think don't need to take in `src` as an arg, as the function can just reference `code` directly. Unless you have another use case haha

Reply:
oh ya okok thanksss